iib/web/api_v1.py

```diff
@@ -155,14 +156,15 @@ def add_bundles():
         payload.get('cnr_token'),
         payload.get('organization'),
         _should_force_overwrite() or payload.get('overwrite_from_index'),
+        flask.current_app.config['IIB_GREENWAVE_CONFIG'].get(celery_queue),
```

Comment:
It might be good to log the values that are used here or in the worker.

Reply:
Did it in `greenwave.py`.